src/data/unexpected_pass/SampleData-jgi_mg_data-colon-dna_sample_name.yaml

```diff
@@ -0,0 +1,20 @@
+jgi_mg_data:
+  - analysis_type:
+      - metagenomics
+      - metatranscriptomics
+    dna_concentration: 1.23
+    dna_cont_type: plate
+    dna_cont_well: C3
+    dna_container_id: xxx
+    dna_dnase: "no"
+    dna_isolate_meth: xxx
+    dna_project_contact: xxx
+    dna_samp_id: xxx
+    dna_sample_format: DNAStable
+    dna_sample_name: DNA:0546789 # contains colon and should not be allowed 'a-z, A-Z, 0-9, - and _ only'
```

Comment:
Thanks. I am going to merge this, but I would like to hear more about how your legal character list was determined.

Reply:
Thanks Mark! For the dna_sample_name, the submission schema documentation - under the LinkML Source section mentions this. It says: "Sample names must be unique across all JGI projects and contain a-z, A-Z, 0-9, - and _ only."

https://microbiomedata.github.io/submission-schema/dna_sample_name/

Reply:
OK, I guess we never implemented that!